server/routes/api/projects.js

```diff
@@ -9,4 +9,7 @@ router.route('/')
 router.route('/:project')
   .get(auth.optional, projectsController.getProject)
 
+router.route('/')
+  .post(projectsController.createProject)
```

Comment:
this `.post(..)` can be chained off the end of the `router.route('/')` above on line 6.
it should look something like
```
router.route('/')
  .get(..)
  .post(..)
```